### PR TITLE
[.Net] Add lookup of the current Namespace before looking up Using

### PR DESCRIPTION
**Pull request by gwenzek**
As discuted in #69 , add the current namespace for looking up the members.

### CODE REVIEW

**Comment by quarnster** [comment]
Could you add one or a couple of test cases hitting this new code also please? Just to make sure it doesn't accidentally break in the future.

File: net/testdata/NamespaceTest.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+namespace System {
+    public class Foo {
+        public void foo(Int32 a){
+            a.
+            Int32 e;
+            e.
+        }
+
+        public void foo(Foo a){
+            a.
```
Comment (gwenzek) [inline net/testdata/NamespaceTest.cs]: #70 this one doesn't work yet, `a` is recognized as `Int32` instead of `Foo`

**Comment by gwenzek** [comment]
I added two tests. I used the `System` namespace because the `System.dll` is always loaded by `completion` so it avoids me to create a test dll.

**Comment by quarnster** [comment]
LGTM, thanks!

File: net/testdata/NamespaceTest.cs (hunk 1 above)
Comment (quarnster) [inline net/testdata/NamespaceTest.cs:10]: Might be because it doesn't find Foo as a type (IIRC types need to be in a compiled dll/exe).

**Pull request merged**